### ISSUE TITLE
Bug: filter images by container name

### ISSUE THREAD

**Issue by will-moore** [ome/omero_search_engine#79]
This query on idr-next is expected to return only 18 images but returns many more from other studies (not idr0146-dominguez-heartlightsheet/experimentA) - see screenshot

```
{
  "resource": "image",
  "query_details": {
    "and_filters": [
      {
        "name": "Organism",
        "operator": "equals",
        "resource": "container",
        "value": "Mus musculus"
      },
      {
        "name": "name",
        "operator": "equals",
        "resource": "container",
        "value": "idr0146-dominguez-heartlightsheet/experimentA"
      }
    ],
    "case_sensitive": false,
    "or_filters": []
  }
}
```

![Screenshot 2023-02-02 at 15 52 52](https://user-images.githubusercontent.com/900055/216378487-8780f2ad-d288-46c3-be47-16b79078a06c.png)

**Comment by khaledk2** [comment]
I think something is not correct in this part of the query.
```
 {
        "name": "Organism",
        "operator": "equals",
        "resource": "container",
        "value": "Mus musculus"
      }

```
the resource should be `image`.

I think this query should not return any results as there is no key called `Organism` for the container, so I think it is still a bug.

**Comment by khaledk2** [comment]
Sorry, Organism  is a container key, please discard  the previous comment

**Comment by khaledk2** [comment]
I will close as it has been fixed on PR #81
